fusedrug/data/tokenizer/modulartokenizer/README.md

```diff
@@ -83,6 +83,10 @@ Additional tokens that are added to the newly created tokenizer are defined in [
 An example of usage of the modular tokenizer is found here: [ModularTokenizer use](https://github.com/BiomedSciAI/fuse-drug/blob/a1b9564eb54b9fe39890645fb5378c13aedde6fb/fusedrug/data/tokenizer/modulartokenizer/test_multi_tokenizer_use.py#L16). It uses the same config as the creation example, and loads the jsons that were saved by the creation code.
 
 The example loads a ready ModularTokenizer stored here: [Word AA + BPE SMILES, path load](https://github.com/BiomedSciAI/fuse-drug/tree/main/fusedrug/data/tokenizer/modulartokenizer/pretrained_tokenizers/modular_AA_SMILES_single_path) using load(), i.e. no specific json paths need to be specified, and the load config is automatically generated by save().
+
+### Adding special tokens:
+An example of adding special tokens to an existing tokenizer is found here: [ModularTokenizer update]()
```

Comment:
Where 🤔 ?

Reply:
Will update and add links to main once merged